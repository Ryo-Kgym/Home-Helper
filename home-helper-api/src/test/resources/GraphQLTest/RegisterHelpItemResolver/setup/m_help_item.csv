id,item_name,help_point,memo
2,テスト項目2,100,これはテストです。
3,テスト項目3,200,これもテストです。
