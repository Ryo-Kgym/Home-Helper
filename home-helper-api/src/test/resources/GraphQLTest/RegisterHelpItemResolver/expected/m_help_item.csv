id,item_name,help_point,memo
1,テスト,200,これはテストです。
2,テスト項目2,100,これはテストです。
3,テスト項目3,200,これもテストです。
